src/lib/jsonParseV2/parseSubscription.cpp

```diff
@@ -65,9 +67,7 @@ using rapidjson::Value;
 static std::string parseAttributeList(ConnectionInfo* ciP, std::vector<std::string>* vec, const Value& attributes);
```

Comment:
As with parseEntitiesVector() and badInput() I undesrtand that parseStringVector() could be used in parseSubscription.cpp instead of parseAttributeList(), isn't it?

I'd prefer to do it in this PR, for not delaying it too much. But if you prefer not doing so, an alternative is to mark is with a FIXME Px.

Reply:
Sure, I can change that as part of the PR. Almost easier than writing an issue about it ...

Reply:
Fixed in 1a49367